stdlib/filepath/src/main/java/org/ballerinalang/stdlib/filepath/Utils.java

```diff
@@ -45,18 +45,38 @@
      * @return      Ballerina error object.
      */
     public static BError getPathError(String reason, Throwable error) {
+        BError bError;
```

Comment:
Shall we simplify this method as follows?
```ballerina
string errorMsg = error != null && error.getMessage() != null ? error.getMessage() : UNKNOWN_MESSAGE;
return getPathError(reason, errorMsg);
```

Reply:
Fixed in bd349a5dc3dedcb64a08b3c49422502898666a7a